dcpy/test/lifecycle/builds/test_plan.py

```diff
@@ -181,6 +182,30 @@ def test_serializing_and_deserializing(self, get_file_types):
         plan.recipe_from_yaml(lock_file)
 
 
+@pytest.mark.usefixtures("file_setup_teardown")
+@pytest.mark.usefixtures("create_buckets")
+@patch("dcpy.connectors.edm.recipes.get_latest_version")
+class TestRecipeVars(TestCase):
+    def test_version_type_var_is_absent(self, get_latest_version):
+        """Ensures VERSION_TYPE is absent in recipe 'vars' attribute when version_type is None."""
+        setup()
```

Comment:
I personally find one-liner `setup` methods like this make the code harder to read. Thoughts about just setting the env var here instead of calling setup?

Reply:
I used the `setup()` fn instead of explicitly setting up an env variable because it already exists in the file and is used across many other tests. 

If it's a preference, I can replace rename the function to something more telling in another PR but I would rather not do it here

Reply:
(and by fix it, I mean either rename the function, or just inline the env var setting.)

Reply:
I prefer to keep PRs focused on their main objective, but I'm happy to address this for you here